<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CUipEditor</class>
 <widget class="QMainWindow" name="CUipEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>554</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>554</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>单位编辑器</string>
  </property>
  <property name="windowIcon">
   <iconset resource="GameDemoEditor.qrc">
    <normaloff>:/icon/res/GameDemoEditor.ico</normaloff>:/icon/res/GameDemoEditor.ico</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <layout class="QGridLayout" name="gridLayout">
        <item row="5" column="1">
         <widget class="QLineEdit" name="txtMinAttackRange"/>
        </item>
        <item row="15" column="0" alignment="Qt::AlignRight">
         <widget class="QLabel" name="label_11">
          <property name="text">
           <string>最大HP:</string>
          </property>
         </widget>
        </item>
        <item row="13" column="0" alignment="Qt::AlignRight">
         <widget class="QLabel" name="label_9">
          <property name="text">
           <string>护甲类型:</string>
          </property>
         </widget>
        </item>
        <item row="8" column="1">
         <widget class="QLineEdit" name="txtProjMoveSpeed"/>
        </item>
        <item row="2" column="0" alignment="Qt::AlignRight">
         <widget class="QLabel" name="label_15">
          <property name="text">
           <string>名称:</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0" alignment="Qt::AlignRight">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>基础移动速度:</string>
          </property>
         </widget>
        </item>
        <item row="13" column="1">
         <widget class="QComboBox" name="comboBoxArmorType"/>
        </item>
        <item row="16" column="1">
         <widget class="QCheckBox" name="chkFixed">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QLineEdit" name="txtBaseMoveSpeed"/>
        </item>
        <item row="15" column="1">
         <widget class="QLineEdit" name="txtMaxHp"/>
        </item>
        <item row="9" column="1">
         <widget class="QLineEdit" name="txtProjScale"/>
        </item>
        <item row="17" column="1">
         <widget class="QLineEdit" name="txtRewardGold"/>
        </item>
        <item row="0" column="0" alignment="Qt::AlignRight">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>单位模板:</string>
          </property>
         </widget>
        </item>
        <item row="8" column="0" alignment="Qt::AlignRight">
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>投射物移动速度:</string>
          </property>
         </widget>
        </item>
        <item row="6" column="1">
         <widget class="QLineEdit" name="txtAttackRange"/>
        </item>
        <item row="7" column="1">
         <widget class="QLineEdit" name="txtHostilityRange"/>
        </item>
        <item row="9" column="0" alignment="Qt::AlignRight">
         <widget class="QLabel" name="label_8">
          <property name="text">
           <string>投射物缩放:</string>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QLineEdit" name="txtBaseAttackInterval"/>
        </item>
        <item row="16" column="0" alignment="Qt::AlignRight">
         <widget class="QLabel" name="label_12">
          <property name="text">
           <string>固定不动的:</string>
          </property>
         </widget>
        </item>
        <item row="7" column="0" alignment="Qt::AlignRight">
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>仇恨产生距离:</string>
          </property>
         </widget>
        </item>
        <item row="17" column="0" alignment="Qt::AlignRight">
         <widget class="QLabel" name="label_13">
          <property name="text">
           <string>死亡金钱奖励:</string>
          </property>
         </widget>
        </item>
        <item row="6" column="0" alignment="Qt::AlignRight">
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>最大攻击距离:</string>
          </property>
         </widget>
        </item>
        <item row="5" column="0" alignment="Qt::AlignRight">
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>最小攻击距离:</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QLineEdit" name="txtName"/>
        </item>
        <item row="4" column="0" alignment="Qt::AlignRight">
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>基础攻击间隔:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="txtIndex">
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="1" column="0" alignment="Qt::AlignRight">
         <widget class="QLabel" name="label_16">
          <property name="text">
           <string>索引:</string>
          </property>
         </widget>
        </item>
        <item row="10" column="0" alignment="Qt::AlignRight">
         <widget class="QLabel" name="label_18">
          <property name="text">
           <string>攻击类型数量:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QComboBox" name="comboBoxOrgUnit"/>
        </item>
        <item row="10" column="1">
         <widget class="QLineEdit" name="txtAttackTypeCount">
          <property name="minimumSize">
           <size>
            <width>180</width>
            <height>0</height>
           </size>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="11" column="1">
         <widget class="QComboBox" name="comboBoxAttackType"/>
        </item>
        <item row="11" column="0" alignment="Qt::AlignRight">
         <widget class="QLabel" name="label_17">
          <property name="text">
           <string>攻击类型:</string>
          </property>
         </widget>
        </item>
        <item row="12" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item alignment="Qt::AlignRight">
           <widget class="QLabel" name="label_19">
            <property name="text">
             <string>基础</string>
            </property>
           </widget>
          </item>
          <item alignment="Qt::AlignRight">
           <widget class="QLabel" name="lblBaseAttack">
            <property name="text">
             <string>类型</string>
            </property>
           </widget>
          </item>
          <item alignment="Qt::AlignRight">
           <widget class="QLabel" name="label_20">
            <property name="text">
             <string>攻击力:</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="14" column="1">
         <widget class="QLineEdit" name="txtBaseArmor"/>
        </item>
        <item row="14" column="0" alignment="Qt::AlignRight">
         <widget class="QLabel" name="label_10">
          <property name="text">
           <string>基础护甲值:</string>
          </property>
         </widget>
        </item>
        <item row="12" column="1">
         <widget class="QLineEdit" name="txtBaseAttack"/>
        </item>
        <item row="18" column="0" alignment="Qt::AlignRight">
         <widget class="QLabel" name="label_14">
          <property name="text">
           <string>死亡经验奖励:</string>
          </property>
         </widget>
        </item>
        <item row="18" column="1">
         <widget class="QLineEdit" name="txtRewardExp"/>
        </item>
        <item row="19" column="0" alignment="Qt::AlignRight">
         <widget class="QLabel" name="label_21">
          <property name="text">
           <string>技能:</string>
          </property>
         </widget>
        </item>
        <item row="19" column="1">
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QComboBox" name="comboBoxSkills"/>
          </item>
          <item>
           <widget class="QToolButton" name="btnSkill">
            <property name="text">
             <string>编辑</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="btnSkillClear">
            <property name="text">
             <string>清空</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <spacer name="verticalSpacer_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="btnAdd">
          <property name="text">
           <string>新增(&amp;A)</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnUpdate">
          <property name="text">
           <string>修改(&amp;M)</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnDel">
          <property name="text">
           <string>删除(&amp;D)</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnClear">
          <property name="text">
           <string>清空(&amp;C)</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QListWidget" name="lstUnits">
        <property name="minimumSize">
         <size>
          <width>180</width>
          <height>0</height>
         </size>
        </property>
        <property name="toolTip">
         <string>双击删除</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QDialogButtonBox" name="buttonBox">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="standardButtons">
       <set>QDialogButtonBox::NoButton</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_F">
    <property name="title">
     <string>文件(&amp;F)</string>
    </property>
    <addaction name="action_N"/>
    <addaction name="action_O"/>
    <addaction name="action_S"/>
    <addaction name="action_A"/>
    <addaction name="separator"/>
    <addaction name="action_X"/>
   </widget>
   <addaction name="menu_F"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action_N">
   <property name="text">
    <string>新建(&amp;N)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="action_O">
   <property name="text">
    <string>打开(&amp;O)...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="action_S">
   <property name="text">
    <string>保存(&amp;S)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="action_A">
   <property name="text">
    <string>另存为(&amp;A)...</string>
   </property>
  </action>
  <action name="action_X">
   <property name="text">
    <string>退出(&amp;X)</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="GameDemoEditor.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>comboBoxAttackType</sender>
   <signal>currentIndexChanged(QString)</signal>
   <receiver>lblBaseAttack</receiver>
   <slot>setText(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>135</x>
     <y>322</y>
    </hint>
    <hint type="destinationlabel">
     <x>50</x>
     <y>344</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
